digitalforms-api/src/main/resources/application.yml

```diff
@@ -0,0 +1,42 @@
+# Digital Forms API Application properties. 
+# Note: Values prefixed with $ are externalized values and must be set via OS Secrets or if running locally 
+# using environmental variables with same name.      
+    
+management:
+  endpoint:
+    beans:
+      cache:
+        # Actuator bean time to live. Inhibits caching of 'No param' endpoints.
+        time-to-live: 10s
+    #Actuator Health Endpoint Configuration    
+    health:
+      show-components: always
+      show-details: always
+  endpoints:
+    web:
+      exposure:
+        # Actuator endpoint exposure
+        include: health
+        
+server:
+  port: 8082
+  servlet:
+    context-path: /digitalforms-viirp/v1
+    
+# VIPS ORDS CLIENT properties
+vips:
+  rest-api:
+    password: ${DIGITALFORMS_VIPSAPI_PASSWORD:}
+    url: ${DIGITALFORMS_VIPSAPI_URL:}
+    username: ${DIGITALFORMS_VIPSAPI_USERNAME:}
```

Comment:
Password, username, and url don't need a blank default value (e.g. ":"). You should be able to remove these. Thanks!

Reply:
Removing the default blank values are actually causing the the dependency injection issue with test properties that you experienced before while building the project with "mvn clean install" locally. Therefore, I suggest keeping them (at least for this PR) but if there is any other workaround for this that you can suggest, I can try that as well.